pdpdmeetup/settings.py

```diff
@@ -57,7 +62,7 @@
 TEMPLATES = (
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [ os.path.join(BASE_DIR, 'templates') ],
+        'DIRS': [os.path.join(BASE_DIR, 'pdpdmeetup/templates')],
```

Comment:
I think this `DIRS` line is no longer necessary, because `templates` will be automatically detected in an app, and `pdpdmeetup` is an app.